### PR TITLE
misc spelling fixes and removal of unused/superfluous code

### PR DESCRIPTION
**Pull request by virdesai**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: src/components/ChannelInner.js [hunk 1]
```diff
@@ -693,14 +690,7 @@ class ChannelInner extends PureComponent {
           enabled={!this.props.disableKeyboardCompatibleView}
         >
           <ChannelContext.Provider value={this.getContext()}>
-            <SuggestionsProvider
-              handleKeyboardAvoidingViewEnabled={(trueOrFalse) => {
```
Comment (DanC5) [inline src/components/ChannelInner.js:697]: don't need this keyboard avoiding view enabler anymore?

File: src/components/ChannelInner.js [hunk 2]
```diff
@@ -155,7 +155,6 @@ class ChannelInner extends PureComponent {
       threadMessages: [],
       threadLoadingMore: false,
       threadHasMore: true,
-      kavEnabled: true,
```
Comment (vishalnarkhede) [inline src/components/ChannelInner.js:158]: remove this from typescript file as well!

Comment (vishalnarkhede) [inline src/components/ChannelInner.js:158]: Please remove this from typescript file as well!!

File: src/components/ChannelInner.js (hunk 1 above)
Comment (virdesai) [reply src/components/ChannelInner.js:697]: nah it wasn't doing anything

**Pull request merged**